tests/test_concentric_neutral_cable.py

```diff
@@ -0,0 +1,217 @@
+import pint
+from numpy import array
+from numpy.testing import assert_array_almost_equal
+
+from carsons.carsons import ConcentricNeutralCarsonsEquations
+from tests.helpers import ConcentricLineModel
+from tests.test_carsons import OHM_PER_MILE_TO_OHM_PER_METER
+
+ureg = pint.UnitRegistry()
+
+feet = ureg.feet
+inches = ureg.inches
+miles = ureg.miles
+ohms = ureg.ohms
+kft = ureg.feet * 1000
+
+
+def test_concentric_neutral_cable():
+    """
+    Validation test against example in Kersting's book.
+    """
+    model = ConcentricNeutralCarsonsEquations(ConcentricLineModel({
```

Comment:
I'm kind of souring on this API. I think its not great that you have to make two objects and then call a class method.
It might make more sense to either

# expose `impedence` as a function, and you do something like
``` python
 import carsons

z_abc = carsons.impedance(ConcentricNeutralCarsonsEquations(ConcentricLineModel(...)))
```
or we expose a function for each type of cable, e.g.

```python
import carsons
z_abc = carsons.concentric_cabe_impedance(ConcentricCableModel(...))
z_abc = carsons.line_impedance(LineModel(...))
```

Reply:
yeah good point! I like the first idea of exposing `imdedance` as a stand-alone function, so the users don't need to know about class method from `ConcentricNeutralCarsonsEquations`.

Reply:
done in 723ff87